openpype/hosts/traypublisher/plugins/publish/collect_simple_instances.py

```diff
@@ -95,6 +104,40 @@ def process(self, instance):
             )
         )
 
+        # Add targeted family to families
+        family = instance.data["family"]
+        creator_attributes = instance.data["creator_attributes"]
+        render_target = creator_attributes["render_target"]
+        instance.data["families"].append(
+            "{}.{}".format(family, render_target)
+        )
+
+        # Add render target specific data
+        if render_target == "farm":
+            # Farm rendering
+            instance.data["toBeRenderedOn"] = "deadline"
+            instance.data["transfer"] = False
+            instance.data["farm"] = True  # to skip integrate
+            if "review" in instance.data["families"]:
+                # to skip ExtractReview locally
```

Comment:
We should maybe skip farm instances in ExtractReview than removing `review` family.

Reply:
I just copied this from other places that do the same, I agree, but it's not my design choice so I don't know why it was done this way with the whole transfer of families in the submission, you tell me